src/web/components/SignInGate/gateDesigns/SignInGatePersonalisedAdCopyVariant2.tsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+
+import { headline, textSans } from '@guardian/src-foundations/typography';
+import { from } from '@guardian/src-foundations/mq';
+import { space, palette, opinion } from '@guardian/src-foundations';
+import { LinkButton } from '@guardian/src-button';
+import { Link } from '@guardian/src-link';
+import { cmp } from '@guardian/consent-management-platform';
+import { trackLink } from '@frontend/web/components/SignInGate/componentEventTracking';
+import { SignInGateProps } from './types';
+
+const signinGate = css`
+    max-width: 617px;
+
+    ${from.desktop} {
+        min-height: 600px;
+    }
+
+    /* This needs to be here because link styles are applied globally to the article body :/ */
+    a {
+        text-decoration: underline;
+        border-bottom: none;
+        color: ${palette.text.primary};
+
+        :hover {
+            border-bottom: none;
+        }
+    }
+`;
+
+const headingStyles = css`
+    ${headline.small({ fontWeight: 'bold' })};
+    border-top: 2px black solid;
+    padding-bottom: 42px;
+
+    ${from.phablet} {
+        padding-right: 160px;
+        ${headline.medium({ fontWeight: 'bold' })};
+    }
+`;
+
+const bodyBold = css`
+    ${textSans.medium({ fontWeight: 'bold' })}
+    border-top: 1px ${palette.line.primary} solid;
+    padding-bottom: 20px;
+    ${from.phablet} {
+        padding-right: 130px;
+    }
+`;
+
+const bodyText = css`
+    ${textSans.medium({ lineHeight: 'regular' })}
+    padding-bottom: ${space[6]}px;
+
+    ${from.phablet} {
+        padding-right: 160px;
+    }
+`;
+
+const signInHeader = css`
+    padding-bottom: 0;
+`;
+
+const actionButtons = css`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin-bottom: 42px;
+
+    > a {
+        margin-right: ${space[9]}px !important;
```

Comment:
is it possible to avoid `!important` using emotion's class composition?

Reply:
These `!importants` are used on all the designs for a few months now. Not going to fix this now so we can get this test out, but the next set of tests are all design/UI changes so this can be looked at then.